src/pages/Dashboard/Modals/SupplyWithdraw/index.tsx

```diff
@@ -182,18 +185,19 @@ export const SupplyWithdrawUi: React.FC<ISupplyWithdrawUiProps & ISupplyWithdraw
 };
 
 const SupplyWithdrawModal: React.FC<ISupplyWithdrawUiProps> = props => {
-  const { asset, ...rest } = props;
-  const { account } = useContext(AuthContext);
+  const { asset, assets, isXvsEnabled, onClose, ...rest } = props;
+  const { account: { address: accountAddress = '' } = {} } = useContext(AuthContext);
+  const { openSuccessfulTransactionModal } = useSuccessfulTransactionModal();
   const { t } = useTranslation();
   const { userTotalBorrowBalance, userTotalBorrowLimit } = useUserMarketInfo({
-    accountAddress: account?.address,
+    accountAddress,
   });
   const { data: vTokenBalance } = useGetVTokenBalance(
-    { account: account?.address || '', vTokenId: asset.id as VTokenId },
-    { enabled: !!account },
+    { account: accountAddress, vTokenId: asset.id as VTokenId },
+    { enabled: !!accountAddress },
```

Comment:
Just to illustrate what I meant on my comment in your other PR where I was saying we should avoid assigning a default empty string to `accountAddress` from within the query hook directly.
Here you could remove the `enabled` option, and it wouldn't be obvious anymore that this query will first execute and pass an empty string to the contract call as the account address, then send another query once the account address becomes defined.